testing/citest/spinnaker_testing/kubernetes_v2_scenario_support.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Kubernetes platform and test support for SpinnakerTestScenario."""
+
+import citest.kube_testing as kube
+from spinnaker_testing.base_scenario_support import BaseScenarioPlatformSupport
+
+
+class KubernetesV2ScenarioSupport(BaseScenarioPlatformSupport):
+  """Provides SpinnakerScenarioSupport for Kubernetes V2."""
+
+  @classmethod
+  def add_commandline_parameters(cls, scenario_class, builder, defaults):
+    """Implements BaseScenarioPlatformSupport interface.
+
+    Args:
+      scenario_class: [class spinnaker_testing.SpinnakerTestScenario]
+      builder: [citest.base.ConfigBindingsBuilder]
+      defaults: [dict] Default binding value overrides.
+         This is used to initialize the default commandline parameters.
+    """
+
+    #
+    # Operation Parameters
+    #
+    builder.add_argument(
+        '--spinnaker_kubernetes_v2_account',
+        default=defaults.get('SPINNAKER_KUBERNETES_V2_ACCOUNT', None),
```

Comment:
Do you want this to be v2 account, especially long term?
In practice these are run in separate processes so names wouldnt conflict with other tests.

Reply:
This is testing v2 accounts only, so I do expect it to be a v2 account long term unless I'm misunderstanding.